app/scripts/controllers/registration.js

```diff
@@ -69,19 +69,12 @@ angular.module('confRegistrationWebApp')
       $location.path('/' + $rootScope.registerMode + '/' + conference.id + '/page/' + $scope.conference.registrationPages[0].id);
     }
 
-    $scope.validateAndGoToNext = function (isValid) {
-      if (isValid) {
-        if (angular.isDefined($scope.nextPage)) {
-          $location.path('/' + $rootScope.registerMode + '/' + conference.id + '/page/' + $scope.nextPage.id);
-        } else {
-          $location.path('/reviewRegistration/' + conference.id).search('regType', null);
-        }
+    $scope.goToNext = function () {
+      $scope.page.viewed = true;  //flag used in show-errors directive to show errors without inputs being touched upon going back to a page
```

Comment:
This needs to also take into account the registrant.  If you register a second person, all the required fields are red right away.